walkthroughs/week-3-server/examples/server-stats/src/main/java/com/google/sps/servlets/ServerStatsServlet.java

```diff
@@ -1,4 +1,4 @@
-// Copyright 2019 Google LLC
+                                                                        // Copyright 2019 Google LLC
```

Comment:
This PR only includes changes to the walkthrough examples?

Reply:
I am so confused. I didn't add any walkthrough files to the commit, only the script.js that I guess didn't show up because there were no changes? I may just collapse this PR.